docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3'
+services:
+    traefik:
+        image: traefik:v2.2
+        ports:
+            - 8082:8082
```

Comment:
Port 8080 is the Web UI

I'm guessing you're running Traefik locally to route traffic elsewhere for development?

http://localhost:8082/api/ proxies to https://drako-test-api.trumpi.co.za
http://localhost:8082/ proxies to http://172.17.0.1:3000

right?

Reply:
Yeah 
- drako-test-api.trumpi.co.za is a deployed version of the backend
- 172.17.0.1 is the IP of the docker host (I could not get this working with the host name)

Reply:
The idea being that `yarn start` listens on port 3000 of the docker host and traefik proxies to that, with `/api` proxying to the deployed backend. Perhaps later the backend location can be configured so that a developer can run the bakcend locally.